lib/fog/aws/parsers/rds/db_cluster_parser.rb

```diff
@@ -31,8 +31,10 @@ def start_element(name, attrs=[])
 
           def end_element(name)
             case name
-            when 'Port', 'Engine', 'Status', 'BackupRetentionPeriod', 'DBSubnetGroup', 'EngineVersion', 'Endpoint', 'DBClusterParameterGroup', 'DBClusterIdentifier', 'PreferredBackupWindow', 'PreferredMaintenanceWindow', 'AllocatedStorage', 'MasterUsername'
+            when 'Port', 'Engine', 'Status', 'BackupRetentionPeriod', 'EngineVersion', 'Endpoint', 'DBClusterParameterGroup', 'DBClusterIdentifier', 'PreferredBackupWindow', 'PreferredMaintenanceWindow', 'AllocatedStorage', 'MasterUsername'
               @db_cluster[name] = value
+            when 'DBSubnetGroup'
+              @db_cluster['DBSubnetGroupName'] = value
```

Comment:
Should this be nested maybe? I wouldn't expect to change the key from what came originally. ie my expectation would be more that this would read `@db_cluster['DBSubnetGroup'] = value`. Does that make sense?